crates/red_knot_python_semantic/resources/mdtest/comparison/non_bool_returns.md

```diff
@@ -23,6 +23,7 @@ from __future__ import annotations
 
 class A:
     def __lt__(self, other) -> A: ...
+    def __gt__(self, other) -> A: ...
```

Comment:
@sharkdp Same here. I'm not sure if the fix is in the "spirit of the test" but it was wrong before ;)